tests/wasm.rs

```diff
@@ -0,0 +1,20 @@
+use app::contract::APP_ID;
+use app::App;
+
+use cw_orch::prelude::*;
+
+// consts for testing
+const ADMIN: &str = "admin";
+
+#[test]
+fn successful_wasm() {
```

Comment:
This puts a hard requirement on them having wasms in this repo, which is something that most projects don't do.

Reply:
No, the CI calls "just wasm" just before running the tests now

Reply:
Then this should probably be feature-flagged so it only runs in CI (when --all-features is set)